Singularity/Singularity/Map/Camera.cs

```diff
@@ -217,30 +220,60 @@ public void KeyPressed(KeyEvent keyEvent)
         {
             var moved = false;
 
-            foreach (var key in keyEvent.CurrentKeys)
+            if (mNeo)
```

Comment:
what exactly is the mNeo for, and why do you use the keys V,I,U,A for camera control in the case that its true?